Makefile

```diff
@@ -149,6 +149,36 @@ install/kubectl-gadget: kubectl-gadget-$(GOHOSTOS)-$(GOHOSTARCH)
 	mkdir -p ~/.local/bin/
 	cp kubectl-gadget-$(GOHOSTOS)-$(GOHOSTARCH) ~/.local/bin/kubectl-gadget
 
+GADGETCTL_TARGETS = \
```

Comment:
Probably; afaik, we're not 100% sure if `gadgetctl` will stay or whether we combine it with `kubectl-gadget`, so I wanted to keep the changes easily reversible and not overcomplicate the code. For the CI workflow it was easier to add than it would be here (at least with my bash script knowledge), so I don't know if it's worth it right now.